docs/release_notes/0.2.0.md

```diff
@@ -0,0 +1,85 @@
+# Release 0.2.0
+
+## Announcement
+
+This release signifies end of initial development cycle of `eksctl`.
+We have built many new features since first minor release (0.1.0)
+in August 2018, and are switching to a new release cadence now.
+Albeit we have originally planned to do a number of things in 0.2.0,
+the plan has evolved. We will start publishing our roadmap in a similar
+form to the [AWS containers roadmap](https://github.com/aws/containers-roadmap/projects/1). 
+
+## Acknowledgment
+
+Weaveworks team would like to sencirely thank all of the who have
+contributed to the project, your work is highly appreciated and
+the project would get thus far without you!
+
+We have received many very useful contributions from so many of
+you, some of which came as code, and some as issue or participation
+on Slack, we thank all of you!
+
+Most active contributors in the last 12 month were:
+
+@richardcase @christopherhein @mumoshu @adamjohnson01 @archisgore @Lazyshot
+@kschumy @toricls @cristian-radu @tedmiston @karinnainiguez @Chabane @tiffanyfay
+@yutachaos @gruebel @jstrachan @marccarre @PaulMaddox @ozzieba @mukaibot @jmcarp
+@gotjosh @philoserf @pawelprazak @nckturner @af12066 @mgalgs @Jeffwan @paulbsch
+@arbourd @cbluth @knorby @gchaincl @mcfedr @overdrive3000 @prageethw @patstrom
+@sdarwin @unguiculus @Yannig @IPyandy @bowlesns @derwasp @mikeroyal @superseb
+@zironycho @dcherman @silviogutierrez @sixth @dresnick-sf @jonk @danielfm
+@denwwer @thapakazi @lucioveloso @callmeradical @manabusakai @justincormack
+@austbot @pdavies011010 @mreferre @flou @polothy @Tyil @cpaika @arun-gupta
+@cdenneen @danielchalef @mrichman @whereisaaron @StevenACoffman @procyclinsur
+@braderhart @aparamon @JasonSwindle @jicowan @olipachi @trondhindenes @milkowski
+@dingn1 @jamesalbert @stevepe-1 @ArseniiPetrovich @jvanzyl @redborian @tkang007
+@elirankon @alexclifford @arielvinas @bnutt @voxxit 
+
+## Bug Fixes Since 0.1.40
```

Comment:
Summary of what is most likely to be here as we know it so far:
- remove support for deprecated Kubernetes version 1.10
- remove `--strorage-class` flag that was only relevant to Kubernetes version 1.10
- add support for `aws eks get-token` as fallback when `aws-iam-authenticator` is missing
- add allows user to specify IOPS they want when `io1` root EBS volume is used
- fix long-standing issue with EBS deletions

Reply:
Maybe instead of just:

> * remove support for deprecated Kubernetes version 1.10

we could have additional context, like suggested [here](https://github.com/weaveworks/eksctl/issues/1026#issue-469180154), e.g.:

> * remove support for deprecated Kubernetes version 1.10 from 22nd July 2019 onwards. See https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html for more information.

or so?

Reply:
All the above added in https://github.com/weaveworks/eksctl/pull/992/commits/3da177702a22e061b490750e8360900972587398